src_python/habitat_sim/utils/settings.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import habitat_sim
+import habitat_sim.agent
+
+default_sim_settings = {
+    "max_frames": 1000,
+    "width": 640,
+    "height": 480,
+    "default_agent": 0,
+    "sensor_height": 1.5,
+    "hfov": 90,
+    "color_sensor": True,  # RGB sensor (default: ON)
+    "semantic_sensor": False,  # semantic sensor (default: OFF)
+    "depth_sensor": False,  # depth sensor (default: OFF)
+    "ortho_rgba_sensor": False,  # Orthographic RGB sensor (default: OFF)
+    "ortho_depth_sensor": False,  # Orthographic depth sensor (default: OFF)
+    "ortho_semantic_sensor": False,  # Orthographic semantic sensor (default: OFF)
+    "fisheye_rgba_sensor": False,
+    "fisheye_depth_sensor": False,
+    "fisheye_semantic_sensor": False,
+    "equirect_rgba_sensor": False,
+    "equirect_depth_sensor": False,
+    "equirect_semantic_sensor": False,
+    "seed": 1,
+    "silent": False,  # do not print log info (default: OFF)
+}
+
+# build SimulatorConfiguration
+def make_cfg(settings):
+    sim_cfg = habitat_sim.SimulatorConfiguration()
+    if "scene_dataset_config_file" in settings:
+        sim_cfg.scene_dataset_config_file = settings["scene_dataset_config_file"]
+    sim_cfg.frustum_culling = settings.get("frustum_culling", False)
+    if "enable_physics" in settings:
+        sim_cfg.enable_physics = settings["enable_physics"]
+    if "physics_config_file" in settings:
+        sim_cfg.physics_config_file = settings["physics_config_file"]
+    if not settings["silent"]:
+        print("sim_cfg.physics_config_file = " + sim_cfg.physics_config_file)
```

Comment:
This should be moved to example make_cfg()

Reply:
Only the if statement concerning `settings["silent"]`, right? Just checking.

Reply:
Yes :+1: